QuantitativeReporting.py

```diff
@@ -385,11 +385,14 @@ def setupSegmentationObservers(self):
       segNode.AddObserver(vtkSegmentationCore.vtkSegmentation.SegmentAdded, self.onSegmentAdded))
 
   def initializeWatchBox(self, node):
-    try:
-      dicomFileName = node.GetStorageNode().GetFileName()
-      self.watchBox.sourceFile = dicomFileName if os.path.exists(dicomFileName) else None
-    except AttributeError:
+    if not node:
+      self.watchBox.sourceFile = None
+      return
+    dicomFileName = slicer.dicomDatabase.fileForInstance(node.GetAttribute("DICOM.instanceUIDs").split(" ")[0])
```

Comment:
If the volume was not loaded from DICOM it wouldn't appear in the volume selector as we defined it earlier.

https://github.com/QIICR/QuantitativeReporting/blob/master/QuantitativeReporting.py#L173